src/main/java/com/metamx/metrics/GarbageCollectors.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2016 Metamarkets Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.metamx.metrics;
+
+import com.metamx.common.logger.Logger;
+
+import java.lang.management.GarbageCollectorMXBean;
+import java.lang.management.ManagementFactory;
+import java.util.ArrayList;
+import java.util.List;
+
+final class GarbageCollectors
```

Comment:
All internal Perf counters not necessary should be presented in JStat. You can check the list of current counters by this small snippet:
```scala
override def main(args: Array[String]): Unit = {
  val bb = Perf.getPerf.attach(0, "r")
  val instr = new PerfInstrumentation(bb)
  val classes = instr.getAllCounters.asScala.map {
    counter =>
      println("%s (%s): %s".format(counter.getName, counter.getClass.getSimpleName, counter.getValue))
      counter.getClass.getSimpleName
  }
  println(classes.distinct)
}
```

and also check them in JVM sources.